package.json

```diff
@@ -56,9 +49,18 @@
     "@storybook/addon-essentials": "7.0.0",
     "@storybook/addon-interactions": "7.0.0",
     "@storybook/addon-links": "7.0.0",
+    "@storybook/blocks": "7.0.0",
+    "@storybook/components": "7.0.0",
+    "@storybook/core-events": "7.0.0",
+    "@storybook/manager": "0.0.0-pr-24676-sha-214a6f84",
```

Comment:
temporarily reference canary release until https://github.com/storybookjs/storybook/pull/24676 has been released

Reply:
Now that Storybook 7.6 has been released, these should be `7.6.0` instead of the canary versions. This was adressed in #62